packages/analytics/src/helpers.test.ts

```diff
@@ -253,7 +265,12 @@ describe('Gtag wrapping functions', () => {
         initPromise2.resolve(); // Resolves second initialization promise.
         await Promise.all([initPromise1, initPromise2]); // Wait for resolution of Promise.all()
 
-        expect((window['dataLayer'] as DataLayer).length).to.equal(1);
+        const dataLayer = window['dataLayer'] as DataLayer;
+        expect(dataLayer.length).to.equal(1);
+        const data = dataLayer[0];
+        expect(data['0']).to.equal('event');
```

Comment:
nit: this works without the quotes around the numbers - `data[0]`. Each item in dataLayer is an `arguments`: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/arguments

Reply:
Done.